src/completions.ts

```diff
@@ -263,7 +269,8 @@ export abstract class CompletionSourceFuse extends CompletionSource {
         For now just displays all scored elements (see threshold in fuse) and
         focus the best match.
     */
-    setStateFromScore(scoredOpts: ScoredOption[], autoselect = false) {
+    setStateFromScore(scoredOpts: ScoredOption[], _?) {
```

Comment:
```suggestion
    setStateFromScore(scoredOpts: ScoredOption[]) {
```

Was just to make refactoring more comfy